backend/controllers/userDetails.test.js

```diff
@@ -18,206 +19,115 @@ describe('user detail factory', () => {
     true,
   )
 
+  const req = {
```

Comment:
I've pulled out common stubs here so that tests can just override the bits that are important in each test, reducing the bloat in the tests and hopefully making it clearer what each test is doing